hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManagerSafeMode.java

```diff
@@ -640,7 +643,17 @@ private void reportStatus(String msg, boolean rightNow) {
    */
   private class SafeModeMonitor implements Runnable {
     /** Interval in msec for checking safe mode. */
-    private static final long RECHECK_INTERVAL = 1000;
+    private long recheckInterval = 1000;
+
+    public SafeModeMonitor() {
+      Configuration conf = new HdfsConfiguration();
```

Comment:
No need to create a new `conf` object, get the conf object passed on from `BlockManagerSafeMode` constructor and initialise the daemon inside the constructor itself.
```
    smmthread = new Daemon(new SafeModeMonitor(conf));
```